code/game/objects/structures/noticeboard.dm

```diff
@@ -1,82 +1,143 @@
+#define MAX_NOTICES 5
+
+/obj/item/mounted/noticeboard
+	name = "notice board"
+	desc = "A board for pinning important notices upon."
+	icon = 'icons/obj/stationobjs.dmi'
+	icon_state = "noticeboard"
+	w_class = WEIGHT_CLASS_BULKY
+
+/obj/item/mounted/noticeboard/do_build(turf/on_wall, mob/user)
+	new /obj/structure/noticeboard(get_turf(user), get_dir(on_wall, user), building = TRUE)
+	qdel(src)
+
 /obj/structure/noticeboard
 	name = "notice board"
 	desc = "A board for pinning important notices upon."
 	icon = 'icons/obj/stationobjs.dmi'
-	icon_state = "nboard00"
+	icon_state = "noticeboard-5"
 	density = FALSE
 	anchored = TRUE
 	max_integrity = 150
 	var/notices = 0
 
+/obj/structure/noticeboard/New(turf/loc, direction, building = FALSE)
+	. = ..()
+	if(building)
+		setDir(direction)
+		set_pixel_offsets_from_dir(-32, 32, -30, 30)
+		update_icon(UPDATE_ICON_STATE)
+
```

Comment:
Curses to you

Reply:
> Curses to you

![image](https://github.com/ParadiseSS13/Paradise/assets/69762909/8a8bb357-9ebb-4a03-9a69-412dfba30f48)